client/start-with/style.scss

```
@@ -0,0 +1,119 @@
+@import "@automattic/components/src/styles/typography";
+
+.layout.is-section-start-with {
+	height: 100%;
+	display: grid;
+	grid-template-rows: max-content 1fr;
+
+	.layout__content {
+		height: 100%;
+		width: 100%;
+		padding: 0;
+		background: #fdfdfd;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+
+
+		.layout__primary,
+		.main {
+			height: 100%;
+			width: 100%;
+			padding: 0;
+		}
+	}
+
+
+	// stylelint-disable declaration-property-unit-allowed-list
+	.container {
+		height: 100%;
+		display: grid;
+		align-items: center;
+		justify-items: center;
+		grid-template-rows: 1fr 128px;
+
+		.content {
+			max-width: 1280px;
+			display: flex;
+			flex-direction: row;
+
+			.left-column {
+				display: flex;
+				flex-direction: column;
+				gap: 46px;
+
+				.title {
+					color: var(--studio-gray-80);
+					font-family: $font-recoleta;
+					font-size: 79px;
+					line-height: 105%;
+				}
+
+				.subtitle {
+					color: #000;
+					font-family: $font-sf-pro-display;
+					font-size: $font-body;
+					font-style: normal;
+					line-height: 150%;
```

```suggestion
					line-height: 162.5%;
```
Very minor change, just to match the design

---

Fixed at https://github.com/Automattic/wp-calypso/pull/91993/commits/d358237aa25b9bbc119ae533f571480c91323d6a